style.css

```diff
@@ -103,3 +108,270 @@ footer h3 {
   width: 25%;
   margin-right: 60%;
 }
+
+#working {
+  margin: 50px 0 57px 0;
+}
```

Comment:
Kindly use the `grid` display property to style the works section, as it is a requirement for this project to use **both** grid and flexbox 🙏.

<img width="756" alt="image" src="https://user-images.githubusercontent.com/37116322/219717166-23e2818f-3e1d-4fa8-80c6-249a6ce030c2.png">

Reply:
Thank you for notifying the error, the CSS grid is added to the main container "id = working" and flexbox is used for individual card class = "project"